pkg/schedule/schedule_test.go

```diff
@@ -30,6 +30,9 @@ func TestFIFOSchedule(t *testing.T) {
 				t.Fatalf("job#%d: got %d, want %d", i, next, i)
 			}
 			next = i + 1
+			if next%2 == 0 { // prevent panic all jobs
+				panic("panic job")
```

Comment:
Could you, please move the 'panic' supression behavior to a separate test-case ?

-schedule job A
- schedule job B (panicking)
- schedule job C

make sure A & C got executed